statsd/options.go

```diff
@@ -49,6 +51,9 @@ type Options struct {
 	SenderQueueSize int
 	// WriteTimeoutUDS is the timeout after which a UDS packet is dropped.
 	WriteTimeoutUDS time.Duration
+	// Telemetry is a set en metrics automatically injected by the client in the
```

Comment:
It should be `Telemetry is a set of metrics`?

Reply:
Indeed!